### DISCUSSION TITLE
GPT says it doesn't have access to my notes

### DISCUSSION THREAD

**Discussion by Hijotonto** [brianpetro/obsidian-smart-connections#385] [Q&A] [answered]
Hello!
I have configured the API and it works for chat, but when i ask something about a note, GPT says things like "i can't see your notes", or "give me the contents of the note". 

I appreciate any help, because i don't know if i'm doing something wrong or i have a supporter key (Where can i buy one?)

Thanks!

**Comment by quelsemme** [comment, marked as answer]
More info on Smart Chat [here](https://wfhbrian.com/introducing-smart-chat-transform-your-obsidian-notes-into-interactive-ai-powered-conversations/)

I thought I saw it explained more clearly elsewhere, but this is a relevant line:
"Note: To trigger a search of your notes, you must use a self-referential pronoun
ex. I, me, my, mine, we, us, our, ours"


The solution is to ask 'Based on my notes' at the beginning of your query, or 'based on the notes in /' (the slash will prompt you to type a folder name) or 'based on [[' (the open brackets, likewise, will prompt you to type a note name)

Without this, it operates as a generic GPT.

**Reply by Hijotonto** [reply]
Thank you very much, quelsemme. Works like a charm, and obviously i didn't understand at first.

**Comment by antonioaraujo664** [comment]
Is it possible to adapt to another language such as Portuguese?
